notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>Any reason we're using the gcloud commands instead of the client library? @Brad for this</p><p>This comment also applies to GCS + BQ resource creation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Using gloud commands is a convention of the template and the other notebooks. </p><p>I will discuss with Brad later on.</p>

Reply:
We decided to stick to template conventions for now, even if in isolation there may be superior alternatives.